setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='esios',
-    version='0.1.9',
+    version='0.1.10',
```

Comment:
We will change the version and tag in the master branch after this merge

Reply:
OK! Rollback done